kowl/values.yaml

```diff
@@ -133,6 +133,9 @@ secret:
     kafka-tls-passphrase: "kafka-tls-passphrase"
     kafka-sasl-password: "kafka-sasl-password"
 
+  extraData: []
+    # foo: "bar"
```

Comment:
I would use a clearer and more meaningful wording here, e.g.: `example-key`: `secret-value`

Reply:
I just removed it because other empty maps do not have examples as well. Do you think an example would be helpful?

Reply:
In my opinion, an example is always helpful :) but if there are no other examples, having no example is good I guess.